firrtl/src/main/scala/firrtl/annotations/JsonProtocol.scala

```diff
@@ -167,13 +168,21 @@ object JsonProtocol extends LazyLogging {
       })
       .distinct
 
-  def serializeTry(annos: Seq[Annotation]): Try[String] = {
-    val tags = getTags(annos)
+  def serializeTry(annos: Seq[Annotation]): Try[String] = serializeTry(annos, new StringWriter).map(_.toString)
+
+  /** Serialize annotations to a [[java.io.Writer]]
+    *
+    * @param annos Annotations to serialize
+    * @param out Writer to which the serialized annotations will be written
+    * @return
+    */
+  def serializeTry[W <: Writer](annos: Iterable[Annotation], out: W): Try[W] = {
```

Comment:
An alternative is to make this `private`. There is then nothing to actually test here (other than seeing if 2GiB of annotations work... 😓). I'm really questioning if this is necessary to make public.

Reply:
I assumed I'd have to call it from Chisel but ended up not needing to. That being said, the current test is in package `firrtlTests` so would have to mess with that. We have already deprecated any import of `package firrtl` (via the compiler plugin) so I'm a bit 🤷‍♀️ about it.